org.eclipse.lemminx/src/main/java/org/eclipse/lemminx/services/format/DOMElementFormatter.java

```diff
@@ -293,7 +293,8 @@ private int formatEndTagElement(DOMElement element, XMLFormattingConstraints par
 			// Remove spaces and indent if the last child is an element, not text
 			// before formatting: <a> example text <b> </b> [space][space]</a>
 			// after formatting: <a> example text <b> </b>\n</a>
-			if ((element.getLastChild().isElement() || element.getLastChild().isComment())
+			if (element.getLastChild() != null 

```

Comment:
To simplify the code, I think it should be better to store element.getLastChild() in a variable.

```java
DOMNode lastChild = element.getLastChild();
if (lastChild  != null && (lastChild .isElement() || lastChild .isComment()) ...
```

Reply:
Thanks @vrubezhny !